<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameUI</class>
 <widget class="QMainWindow" name="GameUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="hitButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HIt Me!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="standButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QLabel" name="bustLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You Busted!
You Lost $200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="loseLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You Lose!
You Lost $200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="winLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You Won!
You Received $200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wagerLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>What is your wager?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="wagerEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doubleButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Double Down!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="splitButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>310</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Split</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>padmaa-Bold.1.1</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>$500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="card1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>backgroundLabel</zorder>
   <zorder>card1</zorder>
   <zorder>card2</zorder>
   <zorder>card3</zorder>
   <zorder>card4</zorder>
   <zorder>card5</zorder>
   <zorder>card6</zorder>
   <zorder>card7</zorder>
   <zorder>card8</zorder>
   <zorder>hitButton</zorder>
   <zorder>standButton</zorder>
   <zorder>bustLabel</zorder>
   <zorder>loseLabel</zorder>
   <zorder>winLabel</zorder>
   <zorder>wagerLabel</zorder>
   <zorder>wagerEdit</zorder>
   <zorder>startButton</zorder>
   <zorder>doubleButton</zorder>
   <zorder>splitButton</zorder>
   <zorder>moneyLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
